src/basic/path-util.h

```diff
@@ -8,6 +8,7 @@
 #include "macro.h"
 #include "string-util.h"
 #include "time-util.h"
+#include "strv.h"
```

Comment:
not that it matters, but this is not ordered alphabetically...

Reply:
Updated.